assets/js/blocks/product-collection/inspector-controls/columns-control.tsx

```diff
@@ -63,33 +55,35 @@ const ColumnsControl = ( props: DisplayLayoutToolbarProps ) => {
 	return showColumnsControl ? (
 		<>
 			<ToolsPanelItem
+				label={ columnsLabel }
 				hasValue={ () =>
-					defaultLayout?.shrinkColumns !== shrinkColumns
+					defaultLayout?.columns !== columns ||
+					defaultLayout?.type !== type
```

Comment:
I don't think the "defaultLayout?.type !== type" condition is necessary. In my view, "defaultLayout?.columns !== columns" should suffice. no?

Reply:
Good catch, I applied your suggestion!